common/display_model/src/main/scala/weco/catalogue/display_model/work/DisplayAbstractRootConcept.scala

```diff
@@ -38,7 +38,17 @@ case object DisplayAbstractConcept extends GetIdentifiers {
     abstractConcept match {
       case Concept(id, label) =>
         DisplayConcept(
-          id = id.maybeCanonicalId.map { _.underlying },
+          id = id.maybeCanonicalId.map {
+            _.underlying
+          },
```

Comment:
minor, not for this PR: I wonder if we need a case class `DisplayCanonicalId` to handle this for us.